addons/spectator/functions/fnc_start.sqf

```diff
@@ -19,4 +19,20 @@ params [["_player", player]];
 
 if (!(local _player)) exitWith {[QGVAR(start), _this, _player] call CBA_fnc_targetEvent};
 
-["Initialize", [player, [], false, true, true, true, true, true, true, true]] call BIS_fnc_EGSpectator;
+// Determine sides available for spectating
+private _whitelistedSides = switch GVAR(sides) do {
+    // Friendly sides spectator
+    case 0: {player call BIS_fnc_friendlySides};
```

Comment:
`player` will not return correct side for dead units, use `playerSide` (also supported by BIS_fnc)
```suggestion
    case 0: {playerSide call BIS_fnc_friendlySides};
```